docs/users.md

```diff
@@ -0,0 +1,93 @@
+# Users
```

Comment:
I have always been a fan of pluralizing them. I think that may be since I am use to writing APIs. It seems like the docs are written based on the model and the endpoints are written well like endpoints :D. 

I was basing the title of this doc off the property name from SDK. 

I think `sdk.users.get` reads better than `sdk.user.get`.

Reply:
plural works for me too